### PR TITLE
Asjo1 sampletest

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: Selenium_Test_Sample.json [hunk 1]
```diff
@@ -0,0 +1,256 @@
+{
```
Comment (ghost) [inline Selenium_Test_Sample.json:1]: Please remove .json

File: config_80.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Please rename this file to config_80.py and adapt it to your needs
```
Comment (ghost) [inline config_80.py:1]: Please remove this file, we have the template

**Comment by ghost** [comment]
Please Jorge, 

remove / close or update this pull request

merci

**Comment by BT-astauder** [comment]
Oudated

**Pull request closed without merging**